app/views/students/_form.html.erb

```diff
@@ -0,0 +1,196 @@
+<%= form_for @student , class: "form-horizontal" do |f| %>
+<% if @student.errors.any? %>
+  <div id="error_explanation">
+    <div class="alert alert-danger">
+      <%= pluralize(@student.errors.count, "error") %> prohibited this student from being saved:
```

Comment:
I have said this many times to change message to 

**There are errors while saving details. Please see inline messages.**